tests/test_code_directory.py

```diff
@@ -102,40 +113,36 @@ def test_match_excluded_precedence_over_included(self, dir_structure):
         self._assert_expected(files, expected)
 
     def test_test_directory_not_excluded(self, dir_structure):
-        expected = ["test_insecure_random.py", "test_make_request.py"]
+        expected = ["something.py", "test_insecure_random.py", "test_make_request.py"]
         files = match_files(
             dir_structure, exclude_paths=["**/samples/**", "**/more_samples/**"]
         )
         self._assert_expected(files, expected)
 
-    def test_include_test_overridden_by_default_excludes(self, mocker):
```

Comment:
These tests are using parent_dir as "." which does not work as expected. We will revisit